mkdocs.yml

```diff
@@ -8,13 +8,13 @@ docs_dir: .
 pages:
 - Home: index.md
 - Getting started with GlusterFS:
-  - Quick start Guide: Quick-Start-Guide/Quickstart.md
+  - Quick Start Guide: Quick-Start-Guide/Quickstart.md
   - Terminologies: Quick-Start-Guide/Terminologies.md
   - Architecture: Quick-Start-Guide/Architecture.md
 - Install Guide:
   - Overview: Install-Guide/Overview.md
   - Common Criteria: Install-Guide/Common_criteria.md
-  - Quick start to Install: Install-Guide/Quick_start.md
+#  - Quick start to Install: Install-Guide/Quick_start.md
```

Comment:
Please remove this entry and also the `Install-Guide/Quick_start.md` file which will not be built.